deps.edn

```diff
@@ -0,0 +1,2 @@
+{:deps
```

Comment:
Can you delete this file?

Reply:
If we want to start the repl outside of Karaf, we need this.